content/docs/pulumi-cloud/access-management/access-tokens.md

```diff
@@ -158,9 +171,16 @@ Navigate to your Pulumi Organization, then:
 1. Select **Teams**.
 1. Select the Pulumi Team you would like to attach the token to.
 1. Scroll to **Access Tokens**.
+1. Select **Create token**, which will open a dialog.
+1. Provide a unique name for this token across your organization. It can be up to 40 characters.
+1. Optionally, you may assign a description for additional context.
+1. Choose an expiration period up of up to two years. You may also choose that the token does not expire.
+1. Select **Create token** in the dialog to create the token.
```

Comment:
Is it weird that these are all `1.`?

Reply:
This is just a handy way of letting the renderer handle the numbering, so the author doesn't have to remember to change it manually when steps are added/removed later.